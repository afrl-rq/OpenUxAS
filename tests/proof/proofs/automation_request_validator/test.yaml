filenames: ["automation_request_validator.ads"]
