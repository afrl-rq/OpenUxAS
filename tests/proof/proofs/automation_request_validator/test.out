spark-containers-parameter_checks.ads:54:19: medium: postcondition might fail, in instantiation at spark-containers-formal-doubly_linked_lists.ads:97, in instantiation at automation_request_validator.ads:89
spark-containers-parameter_checks.ads:59:19: medium: postcondition might fail, in instantiation at spark-containers-formal-doubly_linked_lists.ads:97, in instantiation at automation_request_validator.ads:89
spark-containers-parameter_checks.ads:64:19: medium: postcondition might fail, in instantiation at spark-containers-formal-doubly_linked_lists.ads:97, in instantiation at automation_request_validator.ads:89
